src/app/selectDates/page.tsx

```
@@ -0,0 +1,82 @@
+      <h2 className="text-3xl font-bold mb-10">
+        Select Date
+      </h2>
+
+      <div className="grid gap-6 sm:grid-cols-2 md:grid-cols-3">
+
+        {(dates as string[]).map((date) => {
+
+          const jsDate = new Date(date);
+
+          const today = new Date();
+          const tomorrow = new Date(Date.now() + 86400000);
```

ESLint

[error] Error: Cannot call impure function during render

`Date.now` is an impure function. Calling an impure function can produce unstable results that update unpredictably when the component happens to re-render. (https://react.dev/reference/rules/components-and-hooks-must-be-pure#components-and-hooks-must-be-idempotent).

/home/jailuser/git/src/app/selectDates/page.tsx:49:37
  47 |
  48 |           const today = new Date();
> 49 |           const tomorrow = new Date(Date.now() + 86400000);
     |                                     ^^^^^^^^^^ Cannot call impure function
  50 |
  51 |           const label =
  52 |             jsDate.toDateString() === today.toDateString()

(react-hooks/purity)